### PR TITLE
Database/week3

### PR DESCRIPTION
**Pull request by tiago86g**

### CODE REVIEW

File: databases/week2/School database_DB_w2.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
+CREATE DATABASE school;
+USE school;
+
+-- Create a new database containing the following tables:
+	-- Class: with the columns: id, name, begins (date), ends (date)
+CREATE TABLE `class` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` VARCHAR (180) NOT NULL,
+`begins` DATETIME NOT NULL,
+`ends` DATETIME NOT NULL,
```
Comment (babakbandpey) [inline databases/week2/School database_DB_w2.sql]: the ending comma creates an error

File: databases/week2/School database_DB_w2.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` varchar(255) NOT NULL,
+`email` varchar(255) NOT NULL,
+`phone` varchar(255) NULL,
+`class_id` int(10) unsigned NOT NULL,
+CONSTRAINT `fk_class_id` FOREIGN KEY (`class_id`) REFERENCES `class` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- students
+INSERT INTO student ('id', 'name', 'email', 'phone', 'class_id') VALUES(1, 'Tiago Gonçalves', 'tiago86g@gmail.com', 42594575, 1);
```
Comment (babakbandpey) [inline databases/week2/School database_DB_w2.sql]: All the insert statements need to be changed
The column names may not be enclosed with `'`

INSERT INTO student (`id, name, email, phone, class_id`) VALUES(1, 'Tiago Gonçalves', 'tiago86g@gmail.com', 42594575, 1);

File: databases/week3/book_a_room.sql [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+`id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` VARCHAR(255) NOT NULL,
+`email` VARCHAR(255) NOT NULL UNIQUE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `room` (
+`id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`description` VARCHAR(255) NOT NULL,
+`status_id` INT UNSIGNED NOT NULL ,
+KEY `fk_status` (`status_id`),
```
Comment (babakbandpey) [inline databases/week3/book_a_room.sql]: There is an error in the above line

File: databases/week3/book_a_room.sql [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+-- CONSTRAINT `fk_equipment` FOREIGN KEY (`equipment_id`) REFERENCES `equipment` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `booked_room` (
+`id` INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`startDate` DATETIME NOT NULL,
+`endDate`DATETIME NOT NULL,
+`room_id` INT UNSIGNED NOT NULL, 
+`user_id` INT UNSIGNED NOT NULL, 
+KEY `fk_room` (`room_id`),
```
Comment (babakbandpey) [inline databases/week3/book_a_room.sql]: Above line is incomplete

File: databases/week3/book_a_room.sql [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+KEY `fk_user` (`user_id`),
```
Comment (babakbandpey) [inline databases/week3/book_a_room.sql]: Above line is incomplete

File: databases/week3/meal_application.sql [hunk 6]
```diff
@@ -0,0 +1,33 @@
+CREATE DATABASE meal_application;
+USE meal_application;
+
+CREATE TABLE `Meal` (
+  `id` int,
```
Comment (babakbandpey) [inline databases/week3/meal_application.sql]: `id` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,

File: databases/week3/meal_application.sql [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  `description` text,
+  `location` varchar(255),
+  `when` datetime,
+  `max_reservations` int,
+  `price` decimal,
+  `created_date` date
+);
+
+CREATE TABLE `Reservation` (
+  `id` int,
```
Comment (babakbandpey) [inline databases/week3/meal_application.sql]: `id` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,

File: databases/week3/meal_application.sql [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+  `number_of_guests` int,
+  `meal_id` int,
+  `created_date` date
+);
+
+CREATE TABLE `Review` (
+  `id` int,
```
Comment (babakbandpey) [inline databases/week3/meal_application.sql]: `id` INT NOT NULL AUTO_INCREMENT PRIMARY KEY,

File: databases/week2/School database_DB_w2.sql (hunk 1 above)
Comment (tiago86g) [reply databases/week2/School database_DB_w2.sql]: fixed

File: databases/week2/School database_DB_w2.sql (hunk 2 above)
Comment (tiago86g) [reply databases/week2/School database_DB_w2.sql]: Thank you Babak

File: databases/week3/book_a_room.sql (hunk 4 above)
Comment (tiago86g) [reply databases/week3/book_a_room.sql]: Fixed. Thanks for takings a good look at my work

File: databases/week3/book_a_room.sql (hunk 5 above)
Comment (tiago86g) [reply databases/week3/book_a_room.sql]: Fixed too

File: databases/week3/meal_application.sql (hunk 6 above)
Comment (tiago86g) [reply databases/week3/meal_application.sql]: Fixed. Thank you

File: databases/week3/meal_application.sql (hunk 7 above)
Comment (tiago86g) [reply databases/week3/meal_application.sql]: Fixed. Thank you

File: databases/week3/meal_application.sql (hunk 8 above)
Comment (tiago86g) [reply databases/week3/meal_application.sql]: Fixed. Thank you

**Pull request closed without merging**